apps/web/app/(app)/environments/[environmentId]/components/project-and-org-switch.tsx

```diff
@@ -0,0 +1,81 @@
+"use client";
+
+import { EnvironmentBreadcrumb } from "@/app/(app)/environments/[environmentId]/components/environment-breadcrumb";
+import { OrganizationBreadcrumb } from "@/app/(app)/environments/[environmentId]/components/organization-breadcrumb";
+import { ProjectBreadcrumb } from "@/app/(app)/environments/[environmentId]/components/project-breadcrumb";
+import { Breadcrumb, BreadcrumbList } from "@/modules/ui/components/breadcrumb";
+import { useMemo } from "react";
+import { TEnvironment } from "@formbricks/types/environment";
+import { TOrganization } from "@formbricks/types/organizations";
+import { TProject } from "@formbricks/types/project";
+
+interface ProjectAndOrgSwitchProps {
+  currentOrganization: TOrganization;
+  organizations: TOrganization[];
+  currentProject?: TProject;
+  projects: TProject[];
```

Comment:
We are passing a lot of data from the server through the client components. Can we reduce this? Do we really need the full projects with all attributes or are e.g. project/organization id+name sufficient?

Also we are passing the current organization/project/environment twice, once in the list of all items and also in the current project/organization. Ideally we here only pass the ID and then get this from the list when we need it.